packages/earth-map/src/components/header/component.tsx

```diff
@@ -144,7 +145,7 @@ const Header = (props: IProps) => {
           onClick={handleResetLocation}
         />
       </Link>
-      {allGroups.length > 0 && <>
+      {isAuthenticated && <>
```

Comment:
Please use () and formatting to make code more readable
```suggestion
      {isAuthenticated && (
        <>
          ...
        </>
      )
```